### DISCUSSION TITLE
HeaderFilter and filter row

### DISCUSSION THREAD

**Discussion by Ritzky** [komarovalexander/ka-table#203] [Ideas]
Have the option to use both filter header and filter row

Have the option to only have some columns filter header

**Comment by komarovalexander** [comment]
Hi @Ritzky 

> Have the option to only have some columns filter header

I just released 7.4.1 version which provides the ability to customize headerFilterButton
https://github.com/komarovalexander/ka-table/releases/tag/7.4.1

and I've updated the demo https://komarovalexander.github.io/ka-table/#/header-filter so you can use this solution for your task

as for
> Have the option to use both filter header and filter row

at this time I do not have plans to implement that unfortunately - can you describe the case where you need that?
